ci: re-enable adev tests

JeanMeche opened the pull request.

devversion commented on adev/src/app/features/references/api-reference-details-page/api-reference-details-page.component.spec.ts:
```diff
@@ -68,15 +68,17 @@ describe('ApiReferenceDetailsPage', () => {
   });
 
-  it('should render tabs for all elements with tab attribute', async () => {
+  // TODO: fix & re-enable
```
Nit: do we have an issue for tracking? 😄

devversion replied:
(or should we have one?)

JeanMeche replied:
I went ahead and fixed the tests !

JeanMeche commented:
Yeah let's go with it if the CI is fine !

thePunderWoman commented:
This PR was merged into the repository by commit 7263917793ee57bc83dad148ea109dd007e9bd4d.

The changes were merged into the following branches: main, 19.0.x

thePunderWoman commented:
This seems to have caused real adev test failures at head. I think we need to revert it.

thePunderWoman commented:
Revert is in https://github.com/angular/angular/pull/58592

JeanMeche committed:
ci: re-enable adev tests

Previously we disabled the tests to land some breaking changes.
We can now update the dependencies and re-enable the tests.

thePunderWoman commented:
This PR was merged into the repository by commit 750432337b6df26ddb1497b9d03f1a29f723166e.

The changes were merged into the following branches: main, 19.0.x
